doc/source/language/syntax.rst

```diff
@@ -176,13 +176,13 @@ Help for the new user - What is a Function?
     In programming terminology, there is a commonly used feature of
     many programming languages that works as follows:
 
-    - 1. Create a chunk of program instructions that you don't intend to execute YET.
-    - 2. Later, when executing other parts of the program, do the following:
+    1. Create a chunk of program instructions that you don't intend to execute YET.
+    2. Later, when executing other parts of the program, do the following:
 
-       - 2.A. Remember the current location in the program.
-       - 2.B. Jump to the previously created chunk of code from (1) above.
-       - 2.C. Run the instructions there.
-       - 2.D. Return to where you remembered from (2.A) and continue from there.
+       A. Remember the current location in the program.
+       B. Jump to the previously created chunk of code from (1) above.
+       C. Run the instructions there.
+       D. Return to where you remembered from (2.A) and continue from there.
```

Comment:
Fair enough. I could fix the OL rendering in CSS, but I don't know sphinx well enough. In the meantime, bullets seems like a good workaround.